ext/date/php_date.c

```diff
@@ -2290,8 +2290,8 @@ static HashTable *date_object_get_properties(zval *object) /* {{{ */
 
 				ZSTR_LEN(tmpstr) = snprintf(ZSTR_VAL(tmpstr), sizeof("+05:00"), "%c%02d:%02d",
 					utc_offset < 0 ? '-' : '+',
-					abs(utc_offset / 3600),
-					abs(((utc_offset % 3600) / 60)));
+					abs((int)(utc_offset / 3600)),
```

Comment:
Looks like `dateobj->time->z` is actually an int, so we can just change the var declaration to `int utc_offset` in this case, rather than having the cast here.

Reply:
Exact.